runtime/tutor

```diff
@@ -1186,6 +1186,160 @@ multiple cursors, they won't be uncommented but commented again.
 
 
 
+=================================================================
+=                   12.1 USING MATCH MODE JUMP                  =
+=================================================================
+
+From normal mode, type "m" to enter the match mode. This is
+convenient to work with bracket pairs and the likes, and with
+their content.
+
+There are several actions that can be performed in match mode,
+as indicated by the help pop up. To jump to a matching ([{ pair,
+simply press "mm". For example on the lines below (starting with
+-->), move the cursor in normal mode to (, and press "mm" to jump
+to the matching ). You can do the same on the line below: for example
+move to ], and press "mm" to jump to [ .
```

Comment:
Should we be talking about `matching bracket pair` instead of `matching ([{ pair`? Helix could have other settings for the actual brackets it is counting as a matching pair?

```suggestion
There are several actions that can be performed in match mode,
as indicated by the help pop-up. To jump to a matching ([{ pair,
simply press mm. For example on the lines below (starting with
-->), move the cursor in normal mode to (, and then press mm to jump
to the matching ). You can do the same on the line below: for example
move to ], and press mm to jump to [ .
```